packages/caravan-health/src/privacy.ts

```diff
@@ -0,0 +1,285 @@
+import { BlockchainClient } from "@caravan/clients";
+import type { UTXO, Transaction } from "@caravan/clients/src/client";
```

Comment:
should be possible to import from the module itself. Possible it's because you didn't export the new types in a way that would be built into the final package?